backend/routers/router_expenses.py

```
@@ -0,0 +1,76 @@
+    if not expense or expense.user_id != current_user.id:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Expense not found")
+    
+    await expense_service.delete_expense(db, expense_id)
+    return {"message": "Expense deleted successfully"}
+
+
+@router.get("/category/{category}", response_model=list[ExpenseOut])
+async def get_expenses_by_category(
+    category: str,
+    current_user: Annotated[UserOut, Depends(get_current_user)],
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)